reflectometry/amor_reduction.ipynb

```diff
@@ -0,0 +1,306 @@
+    "The `sample_angle_offset` allows the angular offset of the sample with respect to the horizon to be accounted for."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "separate-alexandria",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sample = AmorData(data_file, \n",
+    "                  reduction_creator=name, \n",
+    "                  data_owner=data_owner, \n",
+    "                  experiment_id=experiment_id, \n",
+    "                  experiment_date=experiment_date, \n",
+    "                  sample_description=sample_description, \n",
+    "                  reduction_file=notebook_file, \n",
+    "                  reduction_creator_affiliation=affiliation,\n",
+    "                  sample_angle_offset=0.04 * sc.units.deg, \n",
+    "                  sample_size=0.1*sc.units.m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "naval-landscape",
+   "metadata": {},
+   "source": [
+    "Some detector and wavelength masking can then be performed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "embedded-birmingham",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample.detector_masking(y_min=0 * sc.units.m, y_max=100e-3 * sc.units.m)\n",
+    "sample.wavelength_masking()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "adjacent-layout",
+   "metadata": {},
+   "source": [
+    "The `AmorReference` class reads the reference supermirror measurement, and will perform the necessary corrections.\n",
+    "For this measurement, no angular offset is required. \n",
+    "Again, detector and wavelength masking is performed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "proud-silly",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reference = AmorReference(reference_file)\n",
+    "reference.detector_masking(y_min=0 * sc.units.m, y_max=100e-3 * sc.units.m)\n",
+    "reference.wavelength_masking()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "substantial-doubt",
+   "metadata": {},
+   "source": [
+    "For the normalisation of the sample, we use the `Normalisation` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "close-mouth",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "norm = Normalisation(sample, reference)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "broke-olympus",
+   "metadata": {},
+   "source": [
+    "With this object, there is the choice to bin in the $\\lambda$/$\\theta$-space or the $q_z$-space. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "textile-moscow",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bins = (np.linspace(2.5, 15, 50), np.linspace(0.6, 1.25, 50))\n",
+    "lambda_theta = norm.wavelength_theta_bin(bins)"
```

Comment:
This interface should be changed (in the future). As it is it negates all the benefits we have from scipp, making things obvious and safeguarded from errors:
- Which of the two refers to what?
- What are the units?

Similar for a number of other functions I think, but we can do that as a next step (but we actually have to do it, and avoid getting carried away by a flood of feature request and pressure to continue implementing new things before consolidaitng what we have).

Reply:
Agreed, ideally this would be replaced with two `sc.linspace` calls 😉 (https://github.com/scipp/ess/pull/15#discussion_r614664720)